src/bootstrap/tool.rs

```diff
@@ -606,17 +607,22 @@ macro_rules! tool_extended {
 
         impl Step for $name {
             type Output = Option<PathBuf>;
-            const DEFAULT: bool = true;
+            const DEFAULT: bool = $stable; // FIXME What does this do?
```

Comment:
DEFAULT-enabled tools means that this step will be run by default (e.g., on x.py build). In this case it's also subject to the default_condition in should_run, though.

Reply:
@Mark-Simulacrum I updated the PR. Does it make sense now?